draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9582,20 +9570,20 @@ Content-Type: text/plain
    data format, since orthogonal technologies deserve orthogonal specification.
 </t>
 <t>
-   Since message parsing (<xref target="message.body"/>) needs to be independent of method
+   Since message parsing (<xref target="message.body"/>) needs to be
+   independent of method
    semantics (aside from responses to HEAD), definitions of new methods
-   cannot change the parsing algorithm or prohibit the presence of a message
-   body on either the request or the response message.
-   Definitions of new methods can specify that only a zero-length message body
+   cannot change the parsing algorithm or prohibit the presence of a payload
+   on either the request or the response message.
+   Definitions of new methods can specify that only a zero-length payload
    is allowed by requiring a Content-Length header field with a value of "0".
 </t>
 <t>
    A new method definition needs to indicate whether it is safe (<xref
    target="safe.methods"/>), idempotent (<xref target="idempotent.methods"/>),
    cacheable (<xref target="cacheable.methods"/>), what
-   semantics are to be associated with the payload body if any is present
-   in the request and what refinements the method makes to header field
-   or status code semantics.
+   semantics are to be associated with the request payload (if any), and what
```

Comment:
```suggestion
   semantics are to be associated with the request payload data (if any), and what
```

Reply:
I think this one is correct as well. The method needs to talk about what its payload means, even if that is just header fields.